diff -rupN xmlm-1.2.0/pkg/build xmlm-1.2.0-fix/pkg/build
--- xmlm-1.2.0/pkg/build	2013-09-06 17:29:13.000000000 +0100
+++ xmlm-1.2.0-fix/pkg/build	2014-07-16 15:06:40.000000000 +0100
@@ -8,6 +8,7 @@ LOC=`dirname $0`
 . $LOC/pkg-builder
 
 NATIVE=$1
+DYNLINK=$2
 
 add lib pkg/META                
 add lib src/xmlm.mli      
diff -rupN xmlm-1.2.0/pkg/pkg-builder xmlm-1.2.0-fix/pkg/pkg-builder
--- xmlm-1.2.0/pkg/pkg-builder	2013-09-06 17:29:13.000000000 +0100
+++ xmlm-1.2.0-fix/pkg/pkg-builder	2014-07-16 15:08:36.000000000 +0100
@@ -35,7 +35,14 @@ add ()
     case $1 in
         lib)
             if [ "$NATIVE" = "true" ]; then 
-                add_lib $SRC $DST
+                if [ "$DYNLINK" = "true" ]; then
+                    add_lib $SRC $DST
+                else
+                    case $2 in
+                        *.cmxs) ;;
+                        *) add_lib $SRC $DST ;;
+                    esac
+                fi
             else
                 case $2 in
                     *.a | *.cmx | *.cmxa | *.cmxs) ;;
